compiler/src/utils/config.re

```diff
@@ -353,17 +360,24 @@ let stdlib_dir =
     ~conv=option_conv(Cmdliner.Arg.string),
     ~doc="Path to the standard library (stdlib) directory",
     ~env="GRAIN_STDLIB",
+    ~digestable=Digestable,
```

Comment:
Could you remind me why the stdlib option would be `Digestable` but include_dirs is not?

Reply:
It doesn't need to be, good catch.